pricing-page-exercise/merve/index.html

```diff
@@ -1,12 +1,126 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML 5 Boilerplate</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-  </body>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML 5 Boilerplate</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" />
+</head>
+
+<body>
+  <header>
+    <div class="left">
+      <span>Company name</span>
+    </div>
+    <div class="right">
+      <ul>
+        <li><a href="#">Features</a></li>
+        <li><a href="#">Enterprise</a></li>
+        <li><a href="#">Support</a></li>
+        <li><a href="#">Pricing</a></li>
+        <li><a href="#" class="signup">Sign up</a></li>
+      </ul>
+    </div>
+  </header>
+  <div class="pricing1">
```

Comment:
It would be good to add all main content of the page (everything except header and footer) to the `main` element